pincer/utils/types.py

```diff
@@ -21,7 +21,9 @@ def __eq__(self, __o: object) -> bool:
         return __o is MISSING
 
     def __hash__(self) -> int:
```

Comment:
`__hash__` should defo return a `int` type.
Maybe checking for `not hash(command)` would help?

Reply:
It does return an int though?

Reply:
Oh true, i'm dumb xD